tests/test_browser_model.py

```diff
@@ -11,17 +10,22 @@ class TestBrowserModel(object):
     def test_new_private_window(self, selenium):
         """Tests opening a new private browsing window via menu"""
         foxpuppet = FoxPuppet(selenium)
+        wm = foxpuppet.window_manager
+        print(wm.windows)
+        assert len(wm.windows) == 1
         foxpuppet.browser.open_window(private=True)
-        assert len(foxpuppet.windows.all) == 2
-        assert foxpuppet.browser.is_private is False
-        foxpuppet.windows.focus(foxpuppet.windows.all[1])
-        assert foxpuppet.browser.is_private is True
+        assert len(wm.windows) == 2
+        wm.windows[1].focus()
```

Comment:
`open_window` should return the window object, so we should be able to do `new_window.focus()` here.